<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TTCutMainWindowForm</class>
 <widget class="QMainWindow" name="TTCutMainWindowForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <property name="horizontalSpacing">
     <number>4</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabFileView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Cut</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>4</number>
        </property>
        <property name="spacing">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="TTCutVideoInfo" name="videoFileInfo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>55</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>55</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>1</width>
            <height>1</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="TTCutOutFrame" name="cutOutFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>4</horstretch>
              <verstretch>8</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>264</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>1</width>
              <height>4</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>1</width>
              <height>260</height>
             </size>
            </property>
            <property name="title">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TTCurrentFrame" name="currentFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>4</horstretch>
              <verstretch>8</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>400</width>
              <height>264</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>1</width>
              <height>4</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>1</width>
              <height>260</height>
             </size>
            </property>
            <property name="title">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="TTStreamNavigator" name="streamNavigator">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>60</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QTabWidget" name="cutListTab">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>4</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>180</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>220</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>1</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>190</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>Cut-Points</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_3">
              <property name="margin">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="TTCutTreeView" name="cutList" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>4</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>Marker</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_8">
              <property name="margin">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="TTMarkerTreeView" name="markerList" native="true"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>240</width>
              <height>180</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>240</width>
              <height>220</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>190</height>
             </size>
            </property>
            <property name="tabPosition">
             <enum>QTabWidget::North</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_5">
             <attribute name="title">
              <string>Navigation</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_4">
              <property name="margin">
               <number>0</number>
              </property>
              <property name="spacing">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="TTCutFrameNavigation" name="navigation" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>1</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Files</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="horizontalSpacing">
         <number>2</number>
        </property>
        <property name="verticalSpacing">
         <number>40</number>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QGroupBox" name="videoList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>4</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Video Files</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <property name="margin">
            <number>4</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="TTVideoTreeView" name="videoFileList" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="audioFileInfo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>6</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Audio Files</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="margin">
            <number>4</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="TTAudioTreeView" name="audioFileList" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="subtitleFileInfo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>4</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Subtitle Files</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <property name="margin">
            <number>4</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="TTSubtitleTreeView" name="subtitleFileList" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSaveCurrentFrame"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../ttcut_mac/ui</normaloff>../../ttcut_mac/ui</iconset>
    </property>
    <widget class="QMenu" name="menuRecentProjects">
     <property name="title">
      <string>Recent Projects</string>
     </property>
    </widget>
    <addaction name="actionOpenVideo"/>
    <addaction name="actionOpenAudio"/>
    <addaction name="actionOpenSubtitle"/>
    <addaction name="separator"/>
    <addaction name="actionFileNew"/>
    <addaction name="actionFileOpen"/>
    <addaction name="actionFileSave"/>
    <addaction name="actionFileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="menuRecentProjects"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAction"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionFileNew">
   <property name="icon">
    <iconset resource="../../ttcut_mac/ui/mainwindow.qrc">
     <normaloff>:/pixmaps/pixmaps/filenew_16.xpm</normaloff>:/pixmaps/pixmaps/filenew_16.xpm</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="iconText">
    <string>New Project</string>
   </property>
   <property name="toolTip">
    <string>New Project</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset resource="../../ttcut_mac/ui/mainwindow.qrc">
     <normaloff>:/pixmaps/pixmaps/fileopen_16.xpm</normaloff>:/pixmaps/pixmaps/fileopen_16.xpm</iconset>
   </property>
   <property name="text">
    <string>Open Project...</string>
   </property>
   <property name="iconText">
    <string>Open Project</string>
   </property>
   <property name="toolTip">
    <string>Open Project file</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="icon">
    <iconset resource="../../ttcut_mac/ui/mainwindow.qrc">
     <normaloff>:/pixmaps/pixmaps/filesave_16.xpm</normaloff>:/pixmaps/pixmaps/filesave_16.xpm</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="iconText">
    <string>Save Project</string>
   </property>
   <property name="toolTip">
    <string>Save Project file</string>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="icon">
    <iconset resource="../../ttcut_mac/ui/mainwindow.qrc">
     <normaloff>:/pixmaps/pixmaps/filesaveas_16.xpm</normaloff>:/pixmaps/pixmaps/filesaveas_16.xpm</iconset>
   </property>
   <property name="text">
    <string>Save Project as...</string>
   </property>
   <property name="iconText">
    <string>Save Project as</string>
   </property>
   <property name="toolTip">
    <string>Save Project under new file name</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../../ttcut_mac/ui/mainwindow.qrc">
     <normaloff>:/pixmaps/pixmaps/exit_16.xpm</normaloff>:/pixmaps/pixmaps/exit_16.xpm</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSaveCurrentFrame">
   <property name="icon">
    <iconset resource="../../ttcut_mac/ui/mainwindow.qrc">
     <normaloff>:/pixmaps/pixmaps/saveimage_16.xpm</normaloff>:/pixmaps/pixmaps/saveimage_16.xpm</iconset>
   </property>
   <property name="text">
    <string>Save current frame...</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../../ttcut_mac/ui/mainwindow.qrc">
     <normaloff>:/pixmaps/pixmaps/settings_16.xpm</normaloff>:/pixmaps/pixmaps/settings_16.xpm</iconset>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpenVideo">
   <property name="icon">
    <iconset resource="../../ttcut_mac/ui/mainwindow.qrc">
     <normaloff>:/pixmaps/pixmaps/fileopen_16.xpm</normaloff>:/pixmaps/pixmaps/fileopen_16.xpm</iconset>
   </property>
   <property name="text">
    <string>Open Video...</string>
   </property>
   <property name="iconText">
    <string>Open Video...</string>
   </property>
   <property name="toolTip">
    <string>Open Video...</string>
   </property>
  </action>
  <action name="actionOpenAudio">
   <property name="icon">
    <iconset resource="../../ttcut_mac/ui/mainwindow.qrc">
     <normaloff>:/pixmaps/pixmaps/fileopen_16.xpm</normaloff>:/pixmaps/pixmaps/fileopen_16.xpm</iconset>
   </property>
   <property name="text">
    <string>Open Audio...</string>
   </property>
   <property name="iconText">
    <string>Open Audio...</string>
   </property>
   <property name="toolTip">
    <string>Open Audio...</string>
   </property>
  </action>
  <action name="actionOpenSubtitle">
   <property name="icon">
    <iconset resource="ui/mainwindow.qrc">
     <normaloff>:/pixmaps/pixmaps/fileopen_16.xpm</normaloff>:/pixmaps/pixmaps/fileopen_16.xpm</iconset>
   </property>
   <property name="text">
    <string>Open Subtitle...</string>
   </property>
   <property name="iconText">
    <string>Open Subtitle...</string>
   </property>
   <property name="toolTip">
    <string>Open Subtitle...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TTStreamNavigator</class>
   <extends>QGroupBox</extends>
   <header>../gui/ttstreamnavigator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TTCurrentFrame</class>
   <extends>QGroupBox</extends>
   <header>../gui/ttcurrentframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TTCutVideoInfo</class>
   <extends>QGroupBox</extends>
   <header>../gui/ttcutvideoinfo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TTCutOutFrame</class>
   <extends>QGroupBox</extends>
   <header>../gui/ttcutoutframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TTCutTreeView</class>
   <extends>QWidget</extends>
   <header>../gui/ttcuttreeview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TTCutFrameNavigation</class>
   <extends>QWidget</extends>
   <header>../gui/ttcutframenavigation.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TTAudioTreeView</class>
   <extends>QWidget</extends>
   <header>../gui/ttaudiotreeview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TTVideoTreeView</class>
   <extends>QWidget</extends>
   <header>../gui/ttvideotreeview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TTMarkerTreeView</class>
   <extends>QWidget</extends>
   <header>../gui/ttmarkertreeview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TTSubtitleTreeView</class>
   <extends>QWidget</extends>
   <header>../gui/ttsubtitletreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../ttcut_mac/ui/mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
